extension/js/common/api/api.ts

```diff
@@ -806,8 +807,8 @@ export class Api {
     apiFcCall: (path: string, vals: Dict<any>, fmt: ReqFmt = 'JSON') => Api.internal.apiCall(Api.fc.url('api'), path, vals, fmt, undefined, { 'api-version': '3' }),
   };
 
-  public static retreiveBlogPosts = async () => {
-    return await Api.ajax({ url: 'https://flowcrypt.com/feed', dataType: 'json' }, Catch.stackTrace());
+  public static retreiveBlogPosts = async (): Promise<R.FcBlogPost[]> => {
+    return Api.ajax({ url: 'https://flowcrypt.com/feed', dataType: 'json' }, Catch.stackTrace());
```

Comment:
Looks good. Here, having or not having await makes no difference, it will behave the same, because it's an async function anyway - it will always return a promise regardless.